peps/pep-0710.rst

```diff
@@ -437,6 +437,17 @@ contain any entries. In such cases, pip does not create any
 is encouraged for consumers to rebuild wheels with a newer version of pip in
 these cases.
 
+uv developers raised a concern about requiring at least one hash in the
+``provenance_url.json`` file since uv does not calculate distribution hashes
+unless explicitly required. However, as requiring at least one hash aids in
+integrity checks for distributions in scenarios involving lock files or when
+identifying distributions as part of SBOMs, the ``provenance_url.json`` file
+mandates the inclusion of at least one hash for the downloaded distribution.
+Installers that do not compute hashes of distributions as part of the
+installation process (e.g., due to performance reasons) can omit creating the
+``provenance_url.json`` file, keeping the mentioned limitations for the
+auditability of Python environments in mind.
```

Comment:
This is the bulk of the paragraph and only two sentences -- can we rephrase? Overlong sentences are often hard to interpret.